src/store/participantsStore.js

```diff
@@ -775,6 +778,17 @@ const actions = {
 				}
 			}
 
+			// Heal unknown sessions from participants request
+			// If session.inCall is null, this is the best attempt to get the data; but in that case
+			// it will be the same for different sessions, otherwise we trust signaling messages
+			const sessionsToUpdate = sessions.filter(session => participant.sessionIds.includes(session.sessionId))
+			for (const session of sessionsToUpdate) {
+				sessionStore.updateSession(session.signalingSessionId, {
+					attendeeId: participant.attendeeId,
+					inCall: (session.inCall === null) ? participant.inCall : session.inCall,
```

Comment:
```suggestion
					inCall: session.inCall ?? participant.inCall,
					
```

even it is 0, it's fine